openverse_catalog/dags/providers/provider_api_scripts/jamendo.py

```diff
@@ -16,328 +16,223 @@
+        >>> base_url = "https://prod-1.storage.jamendo.com/"
+        >>> url = f"{base_url}?trackid=1532771&format=mp31&from=app-devsite"
+        >>> _remove_param_from_url(url, "from")
+        'https://prod-1.storage.jamendo.com/?trackid=1532771&format=mp31'
+        :return: Tuple with main audio file information:
+        - audio_url
+        - duration (in milliseconds)
+        """
+        if (audio_url := data.get("audio")) is None:
+            return None
+        return self._remove_param_from_url(audio_url, "from")
+
+    @staticmethod
+    def _get_creator_data(data):
+        base_url = "https://www.jamendo.com/artist/"
+        if (creator_name := data.get("artist_name")) is None:
+            return None, None
+
+        creator_id = data.get("artist_id")
+        creator_idstr = data.get("artist_idstr")
+        if creator_id is not None and creator_idstr is not None:
+            creator_url = f"{base_url}{creator_id}/{creator_idstr}"
+        else:
+            creator_url = None
+
+        return creator_name.strip(), creator_url
+
+    @staticmethod
+    def _get_metadata(data):
+        stats = data.get("stats", {})
+        metadata = {
+            "lyrics": data.get("lyrics") or None,
+            "release_date": data.get("releasedate"),
+            "downloads": stats.get("rate_download_total", 0),
+            "listens": stats.get("rate_listened_total", 0),
+            "playlists": stats.get("rate_playlisted_total", 0),
+        }
+        return {k: v for k, v in metadata.items() if v is not None}
+
+    @staticmethod
+    def _get_tags(data):
+        tags = []
+
+        musicinfo = data.get("musicinfo", {})
+        if music_type := musicinfo.get("vocalinstrumental"):
             tags.append(music_type)
-        music_gender = musicinfo.get("gender")
-        if music_gender:
+        if music_gender := musicinfo.get("gender"):
             tags.append(music_gender)
-        music_speed = musicinfo.get("speed")
-        if music_speed:
+        if music_speed := musicinfo.get("speed"):
             tags.append(f"speed_{music_speed}")
+
         for tag_name in ["instruments", "vartags"]:
             tag_value = musicinfo.get("tags", {}).get(tag_name)
             if tag_value:
-                tag_value = [_ for _ in tag_value if _ != "undefined"]
-                tags.extend(tag_value)
-    return tags
-
-
-def _get_genres(item):
-    genres = item.get("musicinfo", {}).get("tags", {}).get("genres", None)
-    return genres
+                tags.extend([tag for tag in tag_value if tag != "undefined"])
+
+        return tags
+
+    def get_record_data(self, data):
+        if (foreign_identifier := data.get("id")) is None:
+            return None
+
+        if (foreign_landing_url := data.get("shareurl")) is None:
+            return None
+
+        if (audio_url := self._get_audio_url(data)) is None:
+            return None
+
+        license_url = data.get("license_ccurl")
+        logger.info(license_url)
```

Comment:
Should this and other `logger` calls be at `logger.debug`? It seems like this is at least 5 log lines per record which could be a lot.